tests/Doctrine/Tests/DbalExtensions/Connection.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DbalExtensions;
+
+use Doctrine\Common\EventManager;
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\Connection as BaseConnection;
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Logging\Middleware as LoggingMiddleware;
+
+use function class_exists;
+
+class Connection extends BaseConnection
```

Comment:
It would be possible to avoid inheritance if there was a factory that would create and configure the connection, but that's minor.

Reply:
We do have a factory for that. For now, the wrapper class makes it easier for us to access the logger of a connection. Since we only use that class in tests, I think it's just fine.